test/unit/registry-routes-component.js

```diff
@@ -453,8 +453,7 @@ describe('registry : routes : component', function(){
     });
 
     it('should set response headers', function() {
-      expect(response.headers).to.not.be.null;      
-      expect(response.headers['test-header']).to.equal('test-value');
+      expect(response.headers).to.be.undefined;
```

Comment:
I think this tests were actually broken. The GET shouldn't have headers inside the response so this fixes it.